nic_api/models.py

```diff
@@ -374,8 +397,21 @@ def from_xml(cls, rr):
         id_ = rr.attrib['id'] if 'id' in rr.attrib else None
         name = rr.find('name').text
         idn_name = rr.find('idn-name').text
-        ttl = rr.find('ttl').text
-        txt = [string.text for string in rr.findall('txt/string')]
+        if rr.find('ttl'):
+                ttl = rr.find('ttl').text
+        else:
+                ttl = DEFAULT_TTL
+	txt = [string.text for string in rr.findall('txt/string')]
         if len(txt) == 1:
             txt = txt[0]
         return cls(id_=id_, name=name, idn_name=idn_name, ttl=ttl, txt=txt)
+    
+#stub for SRV
+class SRVRecord(DNSRecord):
```

Comment:
Consider following PEP8 please.

Reply:
You mean the description of the function in triple quotes, or i have more mistakes? I`m beginner in python, sorry!